Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UwpPlayerBuildTools.cs

```diff
@@ -17,473 +17,100 @@ namespace Microsoft.MixedReality.Toolkit.Core.Utilities.Build
     /// </summary>
     public static class UwpPlayerBuildTools
     {
-        // Build configurations. Exactly one of these should be defined for any given build.
-        public const string BuildSymbolDebug = "DEBUG";
-        public const string BuildSymbolRelease = "RELEASE";
-        public const string BuildSymbolMaster = "MASTER";
-
-        /// <summary>
-        /// A method capable of configuring <see cref="BuildInfo"/> settings.
-        /// </summary>
-        /// <param name="toConfigure">The settings to configure.</param>
-        public delegate void BuildInfoConfigurationMethod(ref BuildInfo toConfigure);
-
-        /// <summary>
-        /// Add a handler to this event to override <see cref="BuildInfo"/> defaults before a build.
-        /// </summary>
-        /// <seealso cref="RaiseOverrideBuildDefaults"/>
-        public static event BuildInfoConfigurationMethod OverrideBuildDefaults;
-
-        /// <summary>
-        /// Event triggered when a build starts.
-        /// </summary>
-        public static event Action<BuildInfo> BuildStarted;
-
-        /// <summary>
-        /// Event triggered when a build completes.
-        /// </summary>
-        public static event Action<BuildInfo, BuildReport> BuildCompleted;
-
-        /// <summary>
-        /// Call this method to give other code an opportunity to override <see cref="BuildInfo"/> defaults.
-        /// </summary>
-        /// <param name="toConfigure">The settings to configure.</param>
-        /// <seealso cref="OverrideBuildDefaults"/>
-        public static void RaiseOverrideBuildDefaults(ref BuildInfo toConfigure)
+        private static void ParseBuildCommandLine(ref UwpBuildInfo buildInfo)
```

Comment:
I notice that we no longer have command line switch `WSAUwpSdk`, to set the Windows SDK. Can this be added as it may be needed for CI builds.

Reply:
Gotcha, I'll be sure to add that back.

I think I may have removed it for technical reasons tho as I was having a hard time getting the unity project to properly update the SDK version (while in editor).

Reply:
Yup later on in `UpdateDependenciesElement` there's a problem with generating the appxmanifest that I ran into.

Also one thing to note, the MRTK has a required min sdk version of `10.0.17134.0`.